indy_node/test/state_proof/test_state_proof_for_get_requests.py

```diff
@@ -146,6 +150,120 @@ def test_state_proof_returned_for_get_schema(looper,
     check_valid_proof(result)
 
 
+def test_state_proof_returned_for_get_context(looper,
```

Comment:
Since both `test_state_proof_returned_for_get_context` are in one module, I believe `nodeSet` fixtures is the same as `nodeSetWithOneNodeResponding`, since `nodeSet` is a module-level fixture, and `nodeSetWithOneNodeResponding` extends it (so this is references to the same nodes) to delay requests for all nodes but one.
You may need to move this test into another test module, for example `indy_node/test/state_proof/test_state_multi_proofs_for_get_requests.py`.